### ISSUE TITLE
Backport to 1.21

### ISSUE THREAD

**Issue by MaraSgit** [Draylar/inmis#179]
The newest version is 1.21.1 only, backport would be great 😌

**Comment by twenty41** [comment]
agreed, plz don't skip 1.21, so many mods support it. mojang is pushing out the small updates frequently now, i think landing on the major updates as long-term-support versions will become the norm perhaps

**Comment by Globox1997** [comment]
I disagree, between 1.21 and 1.21.1 is almost no difference.
Most if not all 1.21 mods can play on 1.21.1

**Comment by twenty41** [comment]
pain. best backpack mod skipping major version number

**Comment by MaraSgit** [comment]
> pain. best backpack mod skipping major version number

I agree with you, my modpack is 1.21 and i want to keep it that way. Pain

**Comment by Globox1997** [comment]
If you wanna make it work on 1.21, use this [https://fabricmc.net/wiki/tutorial:dependency_overrides](https://fabricmc.net/wiki/tutorial:dependency_overrides)

**Comment by twenty41** [comment]
Told it to ignore `minecraft` and `fabric-api` dependencies, now it's throwing more errors

https://www.minecraft.net/en-us/article/the-future-of-minecrafts-development

More frequent updates coming confirmed, many 1.21 mods definitely not instantly compatible with 1.21.1

Wont nag you further, just quite the blue balls to not get the most certified S tier backpack mod on the pack I've been building  so selfishly I ask you reconsider

**Comment by Dantich-sfg** [comment]
> Told it to ignore `minecraft` and `fabric-api` dependencies, now it's throwing more errors
> 
> https://www.minecraft.net/en-us/article/the-future-of-minecrafts-development
> 
> More frequent updates coming confirmed, many 1.21 mods definitely not instantly compatible with 1.21.1
> 
> Wont nag you further, just quite the blue balls to not get the most certified S tier backpack mod on the pack I've been building so selfishly I ask you reconsider

Open the latest version of Inmis in 7Zip (rightclick the java file, open with - 7zip). Open file - fabric.mod.json. At the bottom look for 

  "depends": {
    "java": ">=21",
    "minecraft": ">=1.21.1",
    "fabric-api": ">=0.102.1+1.21.1",
    "fabricloader": ">=0.16.2"
  }
}

Change  "minecraft": ">=1.21.1" to  "minecraft": ">=1.21". Save the file, exit. 7Zip will ask you to save inside the archive - click yes. 

And you're done. All works

**Comment by Dreuzz** [comment]
Just made a pull request [here](https://github.com/Draylar/inmis/pull/182). If it doesnt get merged just check out [my repo](https://github.com/Dreuzz/inmis) instead and built it yourself
